pkg/controller/bookkeepercluster/bookie.go

```diff
@@ -113,6 +113,16 @@ func makeBookiePodSpec(bk *v1alpha1.BookkeeperCluster) *corev1.PodSpec {
 			},
 		})
 	}
+	ledgerDirs := "/bk/ledgers"
+	journalDirs := "/bk/journal"
+	if bk.Spec.Options["ledgerDirectories"] != "" {
```

Comment:
User may not provide "ledgerDirectories" in Bookkeeper Options.
So need to add a key exists check in the map, using:
```
if ledgerDirs, ok := bk.Spec.Options["ledgerDirectories"]; ok {
    //ledgerDriectories propery is provided by user, ... so do blah blah blah...
}
```